fedmsg_meta_fedora_infrastructure/tests/buildsys.py

```diff
@@ -310,7 +348,7 @@ class TestKojiBuildStateChangeStartNoOwner(Base):
     }
 
 
-class TestKojiBuildStateChangeFail(Base):
```

Comment:
Not anymore... before there was a test for 'start' and 'fail', but none for 'cancel' or 'complete'.  Now there's a test for 'start' and 'complete', but none for 'cancel' or 'fail'.